.gitignore

```diff
@@ -24,3 +24,4 @@
 /public/.htaccess
 /.env.local*
 /vendor/bundle
+/Gemfile.lock
```

Comment:
@AriettaZ was this intentional? Did we have a discussion about this?

Reply:
We didn't discuss this. I think we should add Gemfile.lock back to the repository because I added this line and forget to delete it.